Dockerfile

```
@@ -0,0 +1,13 @@
+FROM node:18-alpine as build
+
+WORKDIR /app/
+ADD . .
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)